server/src/server.ts

```diff
@@ -58,7 +57,7 @@ function getTemplateRoot(pathStr, textDocument, diagnosticMap) {
 
     let currentPath = pathStr;
 
-    while(currentPath !== '.') {
+    while(currentPath !== '/') {

```

Comment:
Do you know if this works on Windows.

Reply:
Sir, please review the changes that I made for Windows support.